webapp.py

```
@@ -0,0 +1,285 @@
+    return flask.jsonify({
+        "success": True,
+        "statusText": "",
+    }), 200
+
+
+def build_response(**kwargs):
+    # Make status code mandatory
+    if "status" in kwargs:
+        status = kwargs.pop("status")
+    else:
+        raise SyntaxError(f"Function {build_response} called without a status code argument.")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)